{
  "debianbug": 722055,
  "description": "The X509Extension in pyOpenSSL before 0.13.1 does not properly handle a '\\0' character in a domain name in the Subject Alternative Name field of an X.509 certificate, which allows man-in-the-middle attackers to spoof arbitrary SSL servers via a crafted certificate issued by a legitimate Certification Authority.",
  "releases": {
    "bullseye": {
      "fixed_version": "0.13-2.1",
      "repositories": {
        "bullseye": "20.0.1-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "0.13-2.1",
      "repositories": {
        "buster": "19.0.0-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "0.13-2.1",
      "repositories": {
        "sid": "20.0.1-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "0.13-2.1",
      "repositories": {
        "stretch": "16.2.0-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}